ilastik/applets/dataSelection/dataSelectionGui.py

```diff
@@ -408,22 +407,23 @@ def addFileNames(self, paths: Optional[List[Path]], startingLaneNum: Optional[in
         if paths is None:
             paths = []
```

Comment:
```suggestion
        if paths is None:
            return
```

Now that `addLanes` is executed even when `paths is None`, I'd rather this just exits right here instead of continuing with an empty list.

As far as I can trace, it looks like the downstream code is ok with empty lists, but I didn't check whether all `handleNewLanesAdded` implementations are ok with being triggered when no actual new lanes have been added. I also don't really see the point in continuing with an empty input when the function knows there's nothing to be done.

Reply:
I changed the condition to `not paths` because cancelling the ImageFileDialog gives `paths = []`

Reply:
a right ;D